marytts-common/src/main/java/marytts/fst/AlignerTrainer.java

```diff
@@ -44,32 +44,34 @@
  * The algorithm, in its essence, is implemented after a description of levenshtein distance as it can be found in Wikipedia (see
  * the linked revision):
  * 
- * http://en.wikipedia.org/w/index.php?title=Levenshtein_distance&oldid=349201802#Computing_Levenshtein_distance
+ * @see <a
+ *      href="http://en.wikipedia.org/w/index.php?title=Levenshtein_distance&oldid=349201802#Computing_Levenshtein_distance">Computing
+ *      Levenshtein distance</a>
  * 
```

Comment:
`@see` is not for external weblinks! Please refer to the [reference](http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html) and use the correct annotations.